### ISSUE TITLE
Gets stuck on 'Starting replica set...'

### ISSUE THREAD

**Issue by parthnaik** [vkarpov15/run-rs#53]
I have followed the instructions given [here](http://thecodebarbarian.com/introducing-run-rs-zero-config-mongodb-runner.html).

I ran the commands:
`npm install run-rs -g`
`run-rs -v 4.4.0 --shell`

But on the second command, although it downloads Mongo successfully, it gets stuck on `Starting replica set...`

**Comment by parthnaik** [comment]
Setting it to use 4.2.0 worked for me.

**Comment by fabien** [comment]
Version 4.4.1 isn't working for me either.

**Comment by codedavinci** [comment]
@fabien  +1

**Comment by codedavinci** [comment]
Just checked the source code and it hard codes to get a version up to 4.2 ..

**Comment by bhelfert** [comment]
I have tried to run it with my locally installed version 4.4.1, using `run-rs -m`, but the initialization seems to get stuck:
```
> rs.status()
{
	"operationTime" : Timestamp(0, 0),
	"ok" : 0,
	"errmsg" : "no replset config has been received",
	"code" : 94,
	"codeName" : "NotYetInitialized",
	"$clusterTime" : {
		"clusterTime" : Timestamp(0, 0),
		"signature" : {
			"hash" : BinData(0,"AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
			"keyId" : NumberLong(0)
		}
	}
}
```
With 4.2.11 it works.

**Comment by vkarpov15** [comment]
This looks like an issue with [mongodb-topology-manager depending on a MongoDB log message](https://github.com/mongodb-js/mongodb-topology-manager/blob/60f6fb29e3e78d517f84433f1a8cc23902e88aee/lib/server.js#L359-L360) that changed slightly in MongoDB 4.4. There was a [fix for this issue](https://github.com/mongodb-js/mongodb-topology-manager/commit/7b30820c5d7f4fb4c3c748e42e456fb249469803) but it hasn't been published to npm.

Seeing that mongodb-topology-manager hasn't seen a new release in years, I think we may need to fork it. In the short term, we can probably create a quick patch for this.

**Comment by vkarpov15** [comment]
Fix is in v0.7.2, see https://github.com/vkarpov15/run-rs/commit/e50337fa2ae6acfb78ae3db42d71929fb7848734

**Comment by bhelfert** [comment]
Wow, great! It works! Thank you very much for fixing it both in general and so fast :)!!
